Automattic/jetpack
# Jetpack Form: increases AI request counter optimistically

This PR registers the AI Assistant request optimistically when a suggestion is performed from the Jetpack Form with AI Assistant.

Fixes #

## Proposed changes:

* Jetpack Form: increases AI request counter optimistically

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Open the Jetpack sidebar
* Open the Usage panel
* Create a Jetpack Form block instance
* Request an AI suggestion
* Confirm that the app increases the counter once it's done
* Confirm it increases the counter also when the error is prompt is unclear 

https://github.com/Automattic/jetpack/assets/77539/dff04dda-e35d-4339-8720-721fd337f60a
